dbs/mysql/8/Dockerfile

```diff
@@ -1,16 +1,11 @@
 FROM mysql:8
 
 # We use a standard set of database names and users for our sample images
-ENV MYSQL_DATABASE=sample
-ENV MYSQL_USER=metabase
-ENV MYSQL_PASSWORD=metasample123
-
-# MySQL requires a root user password, we just use the same sample password
-ENV MYSQL_ROOT_PASSWORD=metasample123
+ENV MYSQL_DATABASE=sample MYSQL_USER=metabase MYSQL_PASSWORD=metasample123 MYSQL_ROOT_PASSWORD=metasample123
```

Comment:
This is purely a stylistic point (and it's obviously a personal preference) but I find the previous version easier to read.

Reply:
yes, but remember that each line you add to a Dockerfile it generates an intermediate container, making the build process slower! that's why I added all the env variables into just one line. If this is done once in a while we could leave the env variables in separate lines,, but if we are building the containers often then it's a good starting point to cut build time!

Reply:
Oh, I didn't know that. Thank you for the explanation. TIL something new 🤓

Reply:
👋  Just chiming in with historical context. :)

These containers will likely only be built very rarely. The only reason a container would change is if the sample dataset changes. 

One thing I did not get to with this repository was automatic container builds. Right now it's still very unfortunately manual (including a push to Dockerhub), but ideally these would be automated builds that push to GitHub Container Repository.